### PR TITLE
revise sudo config documentation

### PR DESCRIPTION
**Pull request by Christiandike**

### CODE REVIEW

File: docs/source/reference/config-sudo.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ We want to confine these permissions to only what we really need.
 
 ## Edit `/etc/sudoers`
 
-To do this we add to `/etc/sudoers` (use `visudo` for safe editing of sudoers):
+To do this we write to `/etc/sudoers` (use `visudo` for safe editing of sudoers):
```
Comment (minrk) [inline docs/source/reference/config-sudo.md:49]: I think `add` is more appropriate here:

```suggestion
To do this we add to `/etc/sudoers` (use `visudo` for safe editing of sudoers):
```

File: docs/source/reference/config-sudo.md [hunk 2]
```diff
@@ -253,6 +253,6 @@ $ semodule -i sudo_exec_selinux.pp
 ## Troubleshooting: PAM session errors
 
 If the PAM authentication doesn't work and you see errors for
-`login:session-auth`, or similar, considering updating to a more recent version
-of jupyterhub and disabling the opening of PAM sessions with
+`login:session-auth`, or similar, consider updating to a more recent [version
+of jupyterhub](https://readthedocs.org/projects/jupyterhub/) and disabling the opening of PAM sessions with
```
Comment (minrk) [inline docs/source/reference/config-sudo.md:257]: Let's drop this link. It doesn't link to versions of the package.

**Review by minrk — COMMENTED** [review]
Thanks for your contribution! I’ve left some suggestions inline for you to incorporate. You can update this PR with the suggestions and we will come back to review again.

**Comment by Christiandike** [comment]
Thank you for your suggestions @minrk 
I've applied them

**Comment by minrk** [comment]
Great job!

**Pull request merged**